drivers/clock_control/nrf_clock_calibration.c

```diff
@@ -108,7 +108,11 @@ static void cal_lf_callback(struct onoff_manager *mgr,
 static void start_hw_cal(void)
 {
 	/* Workaround for Errata 192 */
-	if (IS_ENABLED(CONFIG_SOC_SERIES_NRF52X)) {
+	if (IS_ENABLED(CONFIG_SOC_NRF52810_QFAA) ||
```

Comment:
well spotted but actually you could just remove whole if. Code is using nrfx driver and workaround is applied there:
https://github.com/zephyrproject-rtos/hal_nordic/blob/1f9145e8c8359f103b4ac5085c6f5d2e66a2557d/nrfx/drivers/src/nrfx_clock.c#L415